yamcs-core/src/main/java/org/yamcs/cascading/YamcsTcLink.java

```diff
@@ -47,6 +46,10 @@ public class YamcsTcLink extends AbstractTcDataLink {
 
     private Set<String> keepUpstreamAcks = new HashSet<>();
 
+/*    private String upstreamCmdName = new String("none");
+    private String upstreamBinaryArgument = new String("none");*/
+
+    private LinkedList<CommandMapData> commandMapDataList = new LinkedList<>();
```

Comment:
why use a linked list instead of an array list?  
linked lists are almost never a good idea in modern computers.